sklearnex/covariance/tests/test_incremental_covariance.py

```diff
@@ -163,6 +168,36 @@ def test_sklearnex_fit_on_random_data(
     assert_allclose(expected_means, result.location_, atol=1e-6)
 
 
+@pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
+def test_whitened_toy_score(dataframe, queue):
+    from sklearnex.covariance import IncrementalEmpiricalCovariance
```

Comment:
just to clarify, this is for validation of addition of score()?

Reply:
Added because of: https://github.com/intel/scikit-learn-intelex/pull/1865#discussion_r1654991351